### PR TITLE
Proofread Use Cases

### PR DESCRIPTION
**Pull request by selfissued**
Correct issues identified while proofreading the Use Cases appendix.

### CODE REVIEW

**Review by peppelinux — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]
looks fine.

**Review by paulbastian — APPROVED** [review]

**Review by bc-pi — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -2257,35 +2257,35 @@ This is a non-exhaustive list of sample use cases.
...
+The End-User intends to acquire a digital criminal record. This involves a visit to the local administration's office to request the official criminal record be issued as a digital Credential. After presenting an ID document, the End-User is prompted to scan a QR code using the Wallet and is informed that the issuance of the Credential will require some time, due to necessary background checks by the authority.
 
 While using the Wallet, the End-User notices an indication that the issuance of the digital record is in progress. After a few days, the End-User receives a notification from the Wallet indicating that the requested Credential was successfully issued. Upon opening the Wallet, the End-User is queried about the download of the Credential. After confirmation, the Wallet fetches and saves the new Credential.
 
-## Wallet Initiated Issuance during Presentation {#use-case-4}
+## Wallet-Initiated Issuance during Presentation {#use-case-4}
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:2272]:
```suggestion
## Wallet-initiated Issuance during Presentation {#use-case-4}
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -2257,35 +2257,35 @@ This is a non-exhaustive list of sample use cases.
...
 
 An End-User comes across a Verifier app that is requesting the End-User to present a Credential, e.g., a driving license. The Wallet determines the requested Credential type(s) from the presentation request and notifies the End-User that there is currently no matching Credential in the Wallet. The Wallet selects a Credential Issuer capable of issuing the missing Credential and, upon End-User consent, sends the End-User to the Credential Issuer's End-User experience (Web site or app). Once authenticated and consent is provided for the issuance of the Credential into the Wallet, the End-User is redirected back to the Wallet. The Wallet informs the End-User that Credential was successfully issued into the Wallet and is ready to be presented to the Verifier app that originally requested presentation of that Credential.
 
-## Wallet Initiated Issuance during Presentation (Requires Presentation of Additional Credentials During Issuance) {#use-case-5}
+## Wallet-Initiated Issuance during Presentation (Requires Presentation of Additional Credentials During Issuance) {#use-case-5}
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:2276]:
```suggestion
## Wallet-initiated Issuance during Presentation (Requires Presentation of Additional Credentials During Issuance) {#use-case-5}
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -2257,35 +2257,35 @@ This is a non-exhaustive list of sample use cases.
...
 
-Upon providing consent, the End-User is sent back to the Wallet. The Wallet informs the End-User Credential was successfully issued into the Wallet and is ready to be presented to the Verifier app that originally requested presentation of that Credential.
+Upon providing consent, the End-User is sent back to the Wallet. The Wallet informs the End-User that the Credential was successfully issued into the Wallet and is ready to be presented to the Verifier app that originally requested presentation of that Credential.
 
-## Wallet Initiated Issuance after Installation {#use-case-6}
+## Wallet-Initiated Issuance after Installation {#use-case-6}
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:2284]:
```suggestion
## Wallet-initiated Issuance after Installation {#use-case-6}
```

**Review by awoie — APPROVED** [review]

**Pull request merged**